src/components/InventoryPage/DonationModal.jsx

```diff
@@ -0,0 +1,405 @@
+  const {
+    id,
+    addressStreet,
+    addressUnit,
+    addressCity,
+    addressZip,
+    firstName,
+    lastName,
+    email,
+    phoneNum,
+    notes,
+    submittedDate,
+    status,
+  } = props;
+
+  const [image, setImage] = useState([]);
+
+  useEffect(async () => {
+    const result = await getPictureFromDB(id);
+    setImage(result[0]);
+    // console.log(result[0]);
+  }, []);
+
+  // console.log(props);
+  // console.log(id);
+
+  const [canEditInfo, setCanEditInfo] = useState(false);
+  const [first, setFirstName] = useState(firstName);
+  // const [last, setLastName] = useState(lastName);
+  const [newemail, setEmail] = useState(email);
+  const [phone, setPhoneNum] = useState(phoneNum);
+  const [street, setStreet] = useState(addressStreet);
+  const [unit, setUnit] = useState(addressUnit);
+  const [city, setCity] = useState(addressCity);
+  const [zip, setZip] = useState(addressZip);
+  const [newnote, setNotes] = useState(notes);
+
+  // const {
+  //   isOpen: donationIsOpen,
+  //   onOpen: donationOnOpen,
+  //   onClose: donationOnClose,
+  // } = useDisclosure();
+  // const { isOpen: imageIsOpen, onOpen: imageOnOpen, onClose: imageOnClose } = useDisclosure();
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  const reloadPage = () => {
+    window.location.reload(true);
+  };
+
+  const updateDonationStatus = async (donationId, newstatus) => {
+    PNPBackend.put(`/donations/${donationId}`, {
+      status: newstatus,
+    }).then(() => {
+      reloadPage();
+    });
+  };
+
+  // const updateFurniture = async () => {
+  //   PNPBackend.put(`/furniture/${furnitureId}`, {
+  //     name: furniturename,
+  //   }).then(() => {
+  //     reloadPage();
+  //   });
+  // };
+
+  const updateDonation = async () => {
+    PNPBackend.put(`/donations/${id}`, {
+      firstName: first,
+      // lastName: last,
+      email: newemail,
+      phoneNum: phone,
+      addressCity: city,
+      addressZip: zip,
+      addressStreet: street,
+      addressUnit: unit,
+      notes: newnote,
+    }).then(() => {
+      reloadPage();
+    });
+  };
+
+  function makeStatus(newStatus) {
+    if (newStatus === 'denied') {
+      return (
+        <Button size="xs" colorScheme="red">
+          REJECTED
+        </Button>
+      );
+    }
+    if (newStatus === 'approved') {
+      return (
+        <Button size="xs" colorScheme="green">
+          APPROVED
+        </Button>
+      );
+    }
+    if (newStatus === 'flagged') {
+      return (
+        <Button size="xs" colorScheme="gray">
+          FLAGGED
+        </Button>
+      );
+    }
+    return (
+      <Button size="xs" colorScheme="gray">
+        {newStatus}
+      </Button>
+    );
+  }
+
+  function makeDate(dateDB) {
+    const d = new Date(dateDB);
+    return `${d.getMonth() + 1}/${d.getDate()}/${d.getFullYear()}`;
+  }
+
+  // console.log(image);
+  return (
+    <Tr onClick={onOpen} key={id}>
+      <Modal isOpen={isOpen} onClose={onClose}>
```

Comment:
The `DonationModal` should be full-sized. Reference the [docs](https://chakra-ui.com/docs/components/modal#modal-sizes) for how to specify sizes.